config/jobs/kubernetes/sig-gcp/sig-gcp-windows.yaml

```diff
@@ -4,7 +4,7 @@ periodics:
   extra_refs:
```

Comment:
@yujuhong do you see anything else in the block below that should be adjusted or pruned?